src/utils/hact/work-order-complete/close-job.test.js

```diff
@@ -15,15 +15,74 @@ describe('buildCloseWorkOrderData', () => {
       },
       jobStatusUpdates: [
         {
-          typeCode: 0,
+          typeCode: '0',
           otherType: 'complete',
           comments: `Work order closed - ${notes}`,
           eventTime: completionDate,
         },
       ],
     }
 
-    const response = buildCloseWorkOrderData(completionDate, notes, reference)
+    const response = buildCloseWorkOrderData(
+      completionDate,
+      notes,
+      reference,
+      'Job Completed'
+    )
+    expect(response).toEqual(CloseWorkOrderFormData)
+  })
+
+  //typeCode 0 for Job Completed
+  it('builds the ScheduleRepair form data to post to the Repairs API with typeCode 0', async () => {
+    const CloseWorkOrderFormData = {
+      workOrderReference: {
+        id: reference,
+        description: '',
+        allocatedBy: '',
+      },
+      jobStatusUpdates: [
+        {
+          typeCode: '0',
+          otherType: 'complete',
+          comments: `Work order closed - ${notes}`,
+          eventTime: completionDate,
+        },
+      ],
+    }
+
+    const response = buildCloseWorkOrderData(
+      completionDate,
+      notes,
+      reference,
+      'Job Completed'
+    )
+    expect(response).toEqual(CloseWorkOrderFormData)
+  })
```

Comment:
Do we need this test? It seems be a duplicate of the one above..

Reply:
yup, made a duplication